sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorItem.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Utils;
+import com.azure.cosmos.util.Beta;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * Change Feed processor item.
+ * Supports current and previous items through {@link JsonNode} structure.
+ *
+ * Caller is recommended to type cast {@link JsonNode} to cosmos item structure.
+ */
+@Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public final class ChangeFeedProcessorItem {
+    @JsonProperty("current")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private JsonNode current;
+    @JsonProperty("previous")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private JsonNode previous;
+    @JsonProperty("metadata")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private ChangeFeedMetaData changeFeedMetaData;
+
+    /**
+     * Gets the change feed current item.
+     *
+     * @return change feed current item.
+     */
+    @Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public JsonNode getCurrent() {
+        return current;
+    }
+
+    /**
+     * Gets the change feed previous item.
+     * For delete operations, previous image is always going to be provided.
+     * The previous image on replace operations is not going to be exposed by default and requires account-level or container-level opt-in.
+     *
+     * @return change feed previous item.
+     */
+    @Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public JsonNode getPrevious() {
+        return previous;
+    }
+
+    /**
+     * Gets the change feed metadata.
+     *
+     * @return change feed metadata.
+     */
+    @Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ChangeFeedMetaData getChangeFeedMetaData() {
+        return changeFeedMetaData;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
How about adding a helper method here - something like -> 

```
public JsonNode toJson() {
 // returns the json node implementation of ChangeFeedProcessorItem.java
}
```

Reply:
This will help customers to re-use their existing use cases of JsonNode handling (like pushing the changes through kafka or any other messaging queue).

Reply:
This is done.